src/rtcModule/webrtc.cpp

```diff
@@ -734,18 +734,12 @@ void Call::handleMessage(RtMessage& packet)
             TermCode reason = static_cast<TermCode>(data.read<uint8_t>(8));
             chatd::EndpointId endPointId(packet.clientid, packet.userid);
 
-            auto itRetriesTime = mSessRetriesTime.find(endPointId);
-            if (!Session::isTermRetriable(reason) && itRetriesTime != mSessRetriesTime.end())
+            auto itRetries = mSessRetries.find(endPointId);
+            if (!Session::isTermRetriable(reason) && itRetries != mSessRetries.end())
```

Comment:
Shouldn't we code it like below for homogeneity?
``` if (!Session::isTermRetriable(reason)
{
    cancelSessionRetryTimer(packet.userid, packet.clientid);
}```

Reply:
I've thought but _destroyIfNoSessionsOrRetries(reason)_  has to execute if both conditions are valid, _!Session::isTermRetriable(reason)_ and _itRetries != mSessRetries.end()_
Then I thought that it is more simple doing without using _cancelSessionRetryTimer_
If you considerate opportune I can change it